galaxy_ng/app/api/v3/viewsets/collection.py

```diff
@@ -315,13 +314,4 @@ def move_content(self, request, *args, **kwargs):
                 base_path=settings.GALAXY_API_DEFAULT_DISTRIBUTION_BASE_PATH
             ).repository
 
-            if dest_repo == golden_repo or src_repo == golden_repo:
-                curate_task = dispatch(
-                    curate_all_synclist_repository,
-                    shared_resources=[golden_repo],
-                    args=(golden_repo.name,),
-                    kwargs={},
-                )
-                response_data['curate_all_synclist_repository_task_id'] = curate_task.pk
-
         return Response(data=response_data, status='202')
```

Comment:
Guess this will result in a likely negligible change to the return results (ie, no 'curate_all_synclist_repository_task_id' since that would no longer make any sense).

Assuming users of the API are okay with that this should be fine.